src/components/pages/ManageResources/RenderResourceManagement.js

```diff
@@ -242,7 +242,32 @@ export const RenderResourceManagement = () => {
         >
           <TextArea />
         </Form.Item>
-        <Form.Item style={{ alignSelf: 'center' }}>
+        <Form.Item
```

Comment:
The purpose of using a `Form.Item` is usually to standardize the look of the app across the whole flow. You can change these in a custom theme component if you want, but usually it doesnt make sense to do it directly on the component itself. Maybe talk to your UX manager and consider changing to give fluidity throughout the app? Just a suggestion

Reply:
I agree here, it may be better to talk with UX manager to make sure that the app looks standardized.